modules/acquia_cms_common/modules/acquia_cms_support/tests/src/Functional/AcquiaCmsConfigSyncTest.php

```diff
@@ -102,21 +102,36 @@ public function testAcquiaConfigSyncOverriddenPage() {
     $assert_session = $this->assertSession();
     $assert_session->statusCodeEquals(200);
 
-    // Asset that overridden configuration appears on overridden tab.
-    $assert_session->elementExists('xpath', "//table/tbody/tr/td[contains(text(),'taxonomy.vocabulary.tags')]");
-    $assert_session->elementExists('xpath', "//table/tbody/tr/td/span[contains(text(),'87  %')]");
+    // Asset that overridden configuration table exist.
+    $assert_session->elementExists('css', 'table.responsive-enabled');
 
-    // Asset that unchanged configuration does not appear on overridden tab.
-    $assert_session->elementNotExists('xpath', "//table/tbody/tr/td[contains(text(),'taxonomy.vocabulary.categories')]");
```

Comment:
This one as well, we are making sure that unchanged configurations appears on unchanged tab here.

Reply:
Done.